src/sidebar/components/menu.js

```diff
@@ -145,15 +145,14 @@ export default function Menu({
       {isOpen && (
         <Fragment>
           {menuArrow(arrowClass)}
-          {/* FIXME-A11Y */}
-          {/* eslint-disable-next-line jsx-a11y/interactive-supports-focus */}
           <div
             className={classnames(
               'menu__content',
               `menu__content--align-${align}`,
               contentClass
             )}
             role="menu"
+            tabIndex="-1"
```

Comment:
The lint error is unambiguous that this element must be focusable, although I notice that the various [menu examples](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html) that I could find don't seem to do this. When a menu is opened, focus is supposed to be placed on the first menu item, and returned to the button when it is closed. It makes sense that the menu items needs to be programatically focusable. I'm not sure about the "menu" itself.

Reply:
I'm not sure we do that with the focus, but I'll add it to he spike for the menu, it would be good to get this stuff right.